dali/operators/reader/loader/webdataset_loader.cc

```diff
@@ -157,8 +157,8 @@ WebdatasetLoader::WebdatasetLoader(const OpSpec& spec)
     }
   }
 
-  dtypes_ = spec.HasArgument("dtypes") ? spec.GetRepeatedArgument<DALIDataType>("dtypes")
-                                       : std::vector<DALIDataType>(ext_.size(), DALI_UINT8);
+  dtypes_ = spec.HasArgument("dtypes") ? spec.GetRepeatedArgument<DALIDataType>("dtypes") :
+                                         std::vector<DALIDataType>(ext_.size(), DALI_UINT8);
```

Comment:
Why? I find it much less readable this way.

Reply:
auto formatter, is it an issue?

Reply:
Well, In general we try not to apply formatting to otherwise unmodified code unless there is an express intention to reformat a piece of code.
Here, the autoformatter destroyed the original intention: aligning the parts of the ternary operator, clearly delineating which part of the expression goes to which case. The dangling colon at the end of the line doesn't read nearly as good.

Reply:
alright